packages/cli/src/types.ts

```diff
@@ -85,10 +85,6 @@ export type Domain = {
   transferredAt?: number | null;
   orderedAt?: number;
   serviceType: 'zeit.world' | 'external' | 'na';
-  verified: boolean;
-  nsVerifiedAt: number | null;
-  txtVerifiedAt: number | null;
-  verificationRecord: string;
```

Comment:
Do we need to bump the API version thats fetched?

Reply:
Bumped 👍